CHANGELOG.md

```diff
@@ -1,5 +1,200 @@
 ## Solidus 2.10.0 (master, unreleased)
 
+### Major Changes
+
+**Added support for Rails 6**
+
+Solidus core now fully support Rails 6! After upgrading to the 2.10 you can
+follow the official Rails Upgrading Guide here:
+https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-5-2-to-rails-6-0
+Please note that Rails 6 requires Ruby 2.5.0 or newer.
+
+- Add support for Rails 6 [#3236](https://github.com/solidusio/solidus/pull/3236) ([aldesantis](https://github.com/aldesantis))
+- Fix dummy app generator to skip Bootsnap and Webpack in Rails 6 [#3327](https://github.com/solidusio/solidus/pull/3327) ([aldesantis](https://github.com/aldesantis))
+- Handle deprecation for Rails 6 in DummyApp [#3352](https://github.com/solidusio/solidus/pull/3352) ([cedum](https://github.com/cedum))
+
+**Deprecated support for Rails 5.1**
+
+Rails 5.1 is deprecated and we'll remove support to 5.1 on the next version.
+If you are still using it, a deprecation warning will be printed in your logs
+when the application boots.
+
+- Deprecate Rails 5.1 [#3333](https://github.com/solidusio/solidus/pull/3333) ([kennyadsl](https://github.com/kennyadsl))
+
+**Changed default images sizes**
+
+We update the images used by Solidus demo in the `sample` gem. To update
+those images we needed to change the default sizes of Spree::Image. They
+changed from:
+
+```
+mini: '48x48>', small: '100x100>', product: '240x240>', large: '600x600>'
+```
+
+to
+
+```
+mini: '48x48>', small: '400x400>', product: '680x680>', large: '1200x1200>'
+```
+
+If your store relies on these sizes, you should change them back following
+the guide here: https://guides.solidus.io/developers/products-and-variants/product-images.html#paperclip-settings.
+
+- Upload new sample images [#3270](https://github.com/solidusio/solidus/pull/3270) ([mfrecchiami](https://github.com/mfrecchiami))
+- Remove unused sample images [#3397](https://github.com/solidusio/solidus/pull/3397) ([JDutil](https://github.com/JDutil))
+- Update sample imgs with wrong file extension [#3343](https://github.com/solidusio/solidus/pull/3343) ([mfrecchiami](https://github.com/mfrecchiami))
+
+**state machines extracted into their own replaceable modules**
+
+This allows stores to replace the state machine completely with their own
+implementatio, even with different gems or using custom code without any
```

Comment:
implementation (missing n)

Nice try to sneak some italian into the changelog, Alberto 😁

Reply:
lol, I think that's more ancient Latin actually. I will fix it up as soon as I can, thanks!

Reply:
Fixed, thanks!